extensions/sql-migration/src/telemetry.ts

```diff
@@ -82,3 +89,14 @@ export function sendSqlMigrationActionEvent(telemetryView: TelemetryViews, telem
 		.withAdditionalMeasurements(additionalMeasurements)
 		.send();
 }
+
+export function getTelemetryProps(migrationStateModel: MigrationStateModel): TelemetryEventProperties {
+	return {
+		'sessionId': migrationStateModel._sessionId,
+		'subscriptionId': migrationStateModel._targetSubscription?.id,
+		'resourceGroup': migrationStateModel._resourceGroup?.name,
+		'targetType': migrationStateModel._targetType,
+		'tenantId': migrationStateModel._azureAccount?.properties?.tenants[0]?.id,
+		'extensionVersion': migrationStateModel.extensionContext.extension.packageJSON.version,
```

Comment:
Extension version is already included in all extension telemetry - you shouldn't need to add it yourself.

Reply:
thanks, will remove. I thought I saw some events where it wasn't emitted, which is why I added her, but think I just missed it.